flake.nix

```diff
@@ -74,8 +74,20 @@
         ${getExe pkgs.reuse} --suppress-deprecation lint
         ${getExe pkgs.lychee} --offline --no-progress .
         ${getExe lint-python}
+        ${getExe format-cpp}
       '';
 
+      format-cpp = pkgs.writeShellApplication {
+        name = "format-cpp";
+        runtimeInputs = [pkgs.clang-tools_18];
+        text = ''
+          set +u
+          ARG="$1"
+          [ -z "$1" ] && ARG="-n"
```

Comment:
I'm not keen on this approach of passing through arguments directly. If people want to use clang-format argument, they can call the tool directly.

If we want to add an easy fix option we could do something similar to what we used to have with sonata-software: https://github.com/lowRISC/sonata-software/blob/206f88d8c81096c46c771ba80bb94c96bc3efa82/flake.nix#L71

Reply:
Done